addons/exact-count/addon.json

```diff
@@ -58,6 +58,8 @@
       "runAtComplete": false
     }
   ],
+  "dynamicEnable": true,
+  "dynamicDisable": true,
```

Comment:
`dynamicDisable` should be `false` here because the userscript doesn't actually support disabling.